### PR TITLE
cross-platform tests

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -39,28 +39,23 @@ jobs:
...
+      - name: Install dependencies and run tests
         run: |
           pip install -e .[test]
           make test-pip
-
-
-
-  
-
+          shell: bash  # Ensure using bash for cross-platform compatibility, especially on Windows
```
Comment (nosheen-adil) [inline .github/workflows/main.yml:61]: @bendichter Looks like the pr checks are failing due to this line. Should look like this instead:
```
shell: bash
run: |
          pip install -e .[test]
          make test-pip
```

**Review by nosheen-adil — CHANGES_REQUESTED** [review]

**Review by nosheen-adil — APPROVED** [review]
@bendichter Thanks for the fix! LGTM

**Comment by nosheen-adil** [comment]
@bendichter Something unusual is happening with the github checks. All the pre merge checks passed, but it still says some checks haven't completed yet, so the PR can't be merged.

I tried re-running the github checks, but no change. Can you push an empty commit to this branch? Maybe that will properly retrigger the checks.

**Comment by CodyCBakerPhD** [comment]
@nosheen-adil this tends to happen when the merge requirements set on the admin side have a mismatch in name - which will happen on this PR since the name of the platform is now shown

You'll probably have to update the branch requirements to match the way they look on this PR (if they show up at all - they may have to first be run on the main fork rather than an outside one), and remove the previous names as requirements

**Pull request merged**